### ISSUE TITLE
Learning rate setting for fine-tuning

### ISSUE THREAD

**Issue by cyh767** [Megvii-BaseDetection/YOLOX#1028]
Hi,

I am fine-tuning YOLOX on COCO and some other datasets for a one-class detection task. I wonder if you could provide some suggestions on the setting of learning rate for fine-tuning? I have set `warmup_epochs = 0`. Shall I keep `basic_lr_per_img = 0.01 / 64.0` or shall I reduce it to a smaller value? Thanks for your help.

**Comment by jeffreyking23** [comment]
I changed the 0.01/64 to 1e-5/64 with batchsize64， and it helped

**Comment by Joker316701882** [comment]
> I changed the 0.01/64 to 1e-5/64 with batchsize64， and it helped

wow, 1e-5/64, this is a very very very low learning rate!

**Comment by Alazaren** [comment]
cyh767, thanks for this insight.
I have noticed that IOU best on the first epoch and drops in further. Using low LR provided by you and looks better now
